koku/reporting/migrations/0099_ocp_performance.py

```diff
@@ -0,0 +1,416 @@
+            reporting_ocpawscostlineitem_project_daily_summary.availability_zone,
+            reporting_ocpawscostlineitem_project_daily_summary.usage_amount,
+            reporting_ocpawscostlineitem_project_daily_summary.unit,
+            reporting_ocpawscostlineitem_project_daily_summary.unblended_cost,
+            reporting_ocpawscostlineitem_project_daily_summary.project_markup_cost,
+            reporting_ocpawscostlineitem_project_daily_summary.pod_cost,
+            reporting_ocpawscostlineitem_project_daily_summary.currency_code
+           FROM reporting_ocpawscostlineitem_project_daily_summary
+          WHERE reporting_ocpawscostlineitem_project_daily_summary.usage_start >= date_trunc('month'::text, date_trunc('month'::text, now()) - '1 day'::interval day)
+        UNION
+         SELECT 'Azure'::text AS source_type,
+            reporting_ocpazurecostlineitem_project_daily_summary.cluster_id,
+            reporting_ocpazurecostlineitem_project_daily_summary.cluster_alias,
+            reporting_ocpazurecostlineitem_project_daily_summary.data_source,
+            reporting_ocpazurecostlineitem_project_daily_summary.namespace,
+            reporting_ocpazurecostlineitem_project_daily_summary.node,
+            reporting_ocpazurecostlineitem_project_daily_summary.pod_labels,
+            reporting_ocpazurecostlineitem_project_daily_summary.resource_id,
+            reporting_ocpazurecostlineitem_project_daily_summary.usage_start::date,
+            reporting_ocpazurecostlineitem_project_daily_summary.usage_end::date,
+            reporting_ocpazurecostlineitem_project_daily_summary.subscription_guid AS usage_account_id,
+            NULL::integer AS account_alias_id,
+            reporting_ocpazurecostlineitem_project_daily_summary.service_name AS product_code,
+            NULL::character varying AS product_family,
+            reporting_ocpazurecostlineitem_project_daily_summary.instance_type,
+            reporting_ocpazurecostlineitem_project_daily_summary.resource_location AS region,
+            NULL::character varying AS availability_zone,
+            reporting_ocpazurecostlineitem_project_daily_summary.usage_quantity AS usage_amount,
+            reporting_ocpazurecostlineitem_project_daily_summary.unit_of_measure AS unit,
+            reporting_ocpazurecostlineitem_project_daily_summary.pretax_cost AS unblended_cost,
+            reporting_ocpazurecostlineitem_project_daily_summary.project_markup_cost,
+            reporting_ocpazurecostlineitem_project_daily_summary.pod_cost,
+            reporting_ocpazurecostlineitem_project_daily_summary.currency AS currency_code
+           FROM reporting_ocpazurecostlineitem_project_daily_summary
+          WHERE reporting_ocpazurecostlineitem_project_daily_summary.usage_start >= date_trunc('month'::text, date_trunc('month'::text, now()) - '1 day'::interval day)) lids
+          with no data;
+
+create index mv_reporting_ocpallcostlineitem_prj_daily_summary_namespace_ix
```

Comment:
This is an extremely silly nitpick, and since this is a migration I really don't care here, but in the future for SQL that we're going to keep in the code base and read, can we CAPS keywords? The mix of CAPS and lower here is confusing to read. Please don't put any effort into changing this PR, just in the future since the existing SQL in the repo uses CAPS, let's do that.

Reply:
OK. It's just a habit since PG will lower everything anyway.